pyproject.toml

```diff
@@ -12,23 +12,24 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.4"
-boto3 = "1.10.46"
-botocore = "1.13.46"
-elasticsearch = "5.5.3"
-aws-requests-auth = "0.4.2"
+python = ">=3.4,<3.7"
+boto3 = "^1.10.46"
+botocore = "^1.13.46"
+# TODO: elasticsearch is on version 7. -kmp 15-Feb-2020
+elasticsearch = "^5.5.3"
```

Comment:
This may have to stay pinned at `5.5.3` for now but it's probably okay to use caret here since it will stay below `6.0.0` where changes will be breaking.

Reply:
Yeah, I figured there could be small bug fixes that snuck in. But the caret is trying to avoid breaking changes for things that respect that. Some of these libraries turn out not to. But I bet they are careful with ES.